hnn_core/gui/_viz_manager.py

```diff
@@ -273,10 +278,11 @@ def _dynamic_rerender(fig):
     fig.tight_layout()
 
 
-def _plot_on_axes(b, widgets_simulation, widgets_plot_type,
-                  target_simulations,
+def _plot_on_axes(b, simulations_widget, widgets_plot_type,
+                  data_widget,
                   spectrogram_colormap_selection, dipole_smooth,
-                  max_spectral_frequency, dipole_scaling, widgets, data,
+                  max_spectral_frequency, dipole_scaling, data_smooth,
+                  data_scaling, widgets, data,
```

Comment:
Could you reorder the keywords so that the `dipole_smooth` and `dipole_scaling` next to one another?

Reply:
Done